response.go

```diff
@@ -11,6 +11,21 @@ func Arguments() []string {
 	return os.Args[1:]
 }
 
+// NormalizedArguments re-normalizes the user arguments provided via Alfred.
+// This isn't necessary for every workflow, specifically only when you're working with special characters.
+// For more info on this topic, please refer to this thread: http://www.alfredforum.com/topic/2015-encoding-issue/
+func NormalizedArguments() (normalizedArgs []string, err error) {
+	for _, e := range Arguments() {
+		var normalizedElement string
+		normalizedElement, err = Normalize(e)
+		if err != nil {
+			return
```

Comment:
Yeah you are right, but let's make it also robust enough for people that don't do error handling :D I changed it in 1735ad2 to use continue, so that not normalized elements are now not part of the result anymore.